dokku-deploy/action.yml

```diff
@@ -0,0 +1,44 @@
+name: Dokku deploy action
+description: Deploy to a Dokku host using GitHub Actions
+inputs:
+  ssh-private-key:
+    description: >
+      The ssh private key to the Dokku host. WARNING: use a secret! 
+      Do not pass a plain text value!
+    required: true
+  ssh-port:
+    description: Custom port for ssh host
```

Comment:
Why does this description differ from the description in README.md? Could you please check all descriptions in this file and make them consistent?

Reply:
fixed